_data/people/cranmer.yml

```
@@ -321,10 +321,24 @@ presentations:
     meetingurl: https://indico.cern.ch/event/962997/
     location: CERN (Virtual)
     focus-area: as
-  - title: How machine learning can help us get the most out of our highest fidelity physical models
-    date: 2020-10-21
-    url: https://www.jlab.org/AI/lunch_series
-    meeting: Jefferson Lab AI Lunch Series
-    meetingurl: https://www.jlab.org/AI/lunch_series
-    location: Jefferson Lab (Virtual)
+  - title: Vision: Physics, Machine Learning, and Computing
```

```suggestion
  - title: "Vision: Physics, Machine Learning, and Computing"
```

@cranmer noticed this `:` wasn't quoted.

---

Good catch Kyle.